python/GafferSceneUI/LightPositionToolUI.py

```diff
@@ -0,0 +1,60 @@
+##########################################################################
+#
+#  Copyright (c) 2023, Cinesite VFX Ltd. All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are
+#  met:
+#
+#      * Redistributions of source code must retain the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer.
+#
+#      * Redistributions in binary form must reproduce the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer in the documentation and/or other materials provided with
+#        the distribution.
+#
+#      * Neither the name of John Haddon nor the names of
+#        any other contributors to this software may be used to endorse or
+#        promote products derived from this software without specific prior
+#        written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+#  IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+#  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+#  PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+#  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+#  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+#  PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+#  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+#  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+#  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+##########################################################################
+
+import imath
+
+import IECore
+
+import Gaffer
+import GafferUI
+import GafferSceneUI
+
+Gaffer.Metadata.registerNode(
+
+	GafferSceneUI.LightPositionTool,
+
+	"description",
+	"""
+	Tool for placing lights.
+	""",
+
+	"viewer:shortCut", "D",
+	"order", 7,
+	"tool:exclusive", True,
+
+	"ui:transformTool:toolTips", IECore.StringVectorData( [ "Hold 'V' to place shadow target", "Hold 'Shift' + 'V' to place shadow pivot", ] ),
```

Comment:
I think it makes sense for the pivot tooltip to go first, since the natural order of operations is to set that first. I'm also wondering if there's a good reason this is StringVectorData (that we join with "\n" later) rather than just being a single string (with `\n` in it)?

Reply:
Right, that was a remnant from when I was doing multiple label widgets. Fixed in https://github.com/GafferHQ/gaffer/pull/5558/commits/6b1ed74ae79ab186f708defeb550512793713967